src/CommonLib/utility.cxx

```diff
@@ -606,4 +567,25 @@ UTILITY::FindDotNetFolders(
     } while (FindNextFileW(handle, &data));
 
     FindClose(handle);
-}
\ No newline at end of file
+}
+
+HANDLE
+UTILITY::CheckIfFileExists(STRU* struFile)
+{
```

Comment:
the name of this function does not match its functionality. If you intent to check file exists, the return value should be bool and you should also handle the scenario of lack access permission

Reply:
> you should also handle the scenario of lack access permission

When you call CreateFile with a lack of access permission, wouldn't that still return INVALID_HANDLE_VALUE?